processing/src/main/java/io/druid/query/groupby/orderby/DefaultLimitSpec.java

```diff
@@ -255,6 +255,13 @@ public int compare(Row left, Row right)
       @Override
```

Comment:
@leventov never mind, I wasnt able to get the `Row` reference inside comparing block in the first pass. But type casted it